java/serve/src/main/java/io/ray/serve/api/ServeControllerClient.java

```diff
@@ -60,16 +69,21 @@ public ServeControllerClient(BaseActorHandle controller, String controllerName)
                 .get();
   }
 
+  public DeploymentHandle getHandle(String deploymentName) {
```

Comment:
should we make it private?

Reply:
It would be a bit challenging to make `ServeControllerClient.getHandle` `private` because it is used in classes like `Serve` and `Deployment` (unable to access `private` methods of `ServeControllerClient`). I will need to reconsider this.